Sources/ManageSettings.php

```diff
@@ -227,6 +228,7 @@ function ModifyBasicSettings($return_config = false)
 		'',
 
 		// Option-ish things... miscellaneous sorta.
+		array('check', 'notify_tokens', 'disabled' => 1),
```

Comment:
what's this setting for ? it doesn't have a label in the `Features and Options` admin page, but the disabled checkbox can be seen.

Reply:
Correct...  This needs to go.

It's a setting in 2.0 that doesn't apply to 2.1.  (Whether to use no-logon-necessary unsub tokens on emails...)